### ISSUE TITLE
Feature request: button to select between boards

### ISSUE THREAD

**Issue by smiska** [ZenHubIO/support#1138]
If you group repositories into more than one board (or into several boards) then it becomes hard to follow which board you are on, what repositories belong to it. It would be great if you could name the different boards and have a button to switch to the other boards. What do you think about it? Is it feasible in the near future? Thanks.

**Comment by kendramoroz** [comment]
@smiska great feature idea! Workspaces and handling better groupings of Boards in a more familiar and focused way is actually something we're currently working on as we build out to launching and enhancing #156 / #1076! 

Bumping this over to discussion so it doesn't get lost until this becomes available 🎉
